### ISSUE TITLE
Is there a way to query executors for a label

### ISSUE THREAD

**Issue by gaborcsardi** [silas/node-jenkins#69]
E.g. this API endpoint works in general: 
```
http://<jenkins-url>/label/<label>/api/json
```
and returns this:
```json
{
  "_class" : "hudson.model.labels.LabelAtom",
  "actions" : [
    
  ],
  "busyExecutors" : 0,
  "clouds" : [
    
  ],
  "description" : null,
  "idleExecutors" : 4,
  "loadStatistics" : {
    "_class" : "hudson.model.Label$1"
  },
  "name" : "rtools3",
  "nodes" : [
    {
      "_class" : "hudson.plugins.swarm.SwarmSlave",
      "nodeName" : "windows-server3-cdf9107c"
    }
  ],
  "offline" : false,
  "tiedJobs" : [
    
  ],
  "totalExecutors" : 4,
  "propertiesList" : [
    
  ]
}
```
and adding `depth=...` will return even more. Do you think there is a way to access this from the package?

**Comment by gaborcsardi** [comment]
Maybe we could have a `jenkins.label.get()` function?

**Comment by gaborcsardi** [comment]
Niiiiiice! Thanks!

**Comment by silas** [comment]
@gaborcsardi fyi: released to npm in `jenkins@0.25.0`

Thanks for letting me know about that endpoint, didn't know it existed.
